packages/logger/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export { Logger } from "./logger";
+export { LoggerProps } from "./types";
```

Comment:
```suggestion
export { LoggerProps, Event } from "./types";
```
and remove the line 3